tests/threading-fpu/src/main.rs

```diff
@@ -0,0 +1,45 @@
+#![no_main]
+#![no_std]
+
+use ariel_os::debug::log::*;
+use ariel_os::debug::{ExitCode, exit, log::info};
+use ariel_os::gpio::{Level, Output};
+
+use ariel_os::thread;
+
+#[ariel_os::thread(autostart)]
+fn thread0() {
+    info!("Hello from thread 0");
+    let mut a: f32 = 1.111;
+    let mut b = 2.222;
+    let mut test = 0.0;
+    for _ in 0..10 {
```

Comment:
Do these tests need some [blackbox](https://doc.rust-lang.org/beta/core/hint/fn.black_box.html) hint? Seems to me the compiler might just pre-calculate the result.

Reply:
Looking at the disassembly I do see the loop so I dont think `blackbox` is required

Reply:
ok!